pyboy/pyboy.py

```diff
@@ -221,6 +222,34 @@ def botsupport_manager(self):
             The manager, which gives easier access to the emulated game through the classes in `pyboy.botsupport`.
         """
         return botsupport.BotSupportManager(self, self.mb)
+    
+    def openai_gymboy(self, observation_type='tiles', buttons_press_mode='press', simultaneous_actions=False):
+        """
+        For Reinforcement learning, it is often easier to use the standard gym environment. This method will provide one.
+        The pyboy need a game_wrapper with a fitness function in order to build a reward system.
```

Comment:
> The pyboy need a game_wrapper with a fitness function in order to build a reward system.

Maybe change this to something like: "This function requires PyBoy to implement a Game Wrapper for the loaded ROM. You can find the supported games in `pyboy.plugins`."

Reply:
Done